__tests__/app.test.js

```diff
@@ -129,3 +129,40 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("returns an array of comments for the given article_id", () => {
+        return request(app).get('/api/articles/1/comments').expect(200).then(({body: {comments}}) => {
+            expect(comments.length).toBe(11)
+        })
+    })
+    test("comments should be formatted correctly", () => {
+        return request(app).get('/api/articles/1/comments').expect(200).then(({body: {comments}}) => {
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
```

Comment:
We should check that the article id specific since we are getting the comments for a specific article